ocs_ci/ocs/resources/mcg.py

```diff
@@ -11,6 +11,11 @@
 from ocs_ci.framework import config
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.bucket_utils import retrieve_verification_mode
+from ocs_ci.ocs.constants import (
+    NODE_READY,
```

Comment:
Create a new `READY_STATE` constant, the node constant might change one day, and then it won't reflect what we're actually looking for